File: sample/C#/CSharp/Kabusapi_Symbolname_Option.cs
using System;
using Newtonsoft.Json;
using System.Net.Http;
using System.Web;

namespace CSharp_sample
{
    class Kabusapi_Symbolname_Option
    {
        static void Main(string[] args)
        {
            string PutOrCall = "C";
            int DerivMonth = 0;
            int StrikePrice = 24000;

            string token = GenerateToken.GetToken();

            var builder = new UriBuilder("http://localhost:18080/kabusapi/symbolname/option");
            var param = HttpUtility.ParseQueryString(builder.Query);
            param["DerivMonth"] = DerivMonth.ToString();
            param["PutOrCall"] = PutOrCall;
            param["StrikePrice"] = StrikePrice.ToString();

            builder.Query = param.ToString();
            string url = builder.ToString();
            try
            {
                var client = new HttpClient();
                var request = new HttpRequestMessage(HttpMethod.Get, url);
                request.Headers.Add("X-API-KEY", token);
                HttpResponseMessage response = client.SendAsync(request).Result;
                Console.WriteLine("{0} \n {1}", JsonConvert.DeserializeObject(response.Content.ReadAsStringAsync().Result), response.Headers);

            }
            catch (HttpRequestException e)
            {
                Console.WriteLine("{0} {1}", e, e.Message);
                Console.ReadKey();
            }
            catch (Exception ex)
            {
                Console.WriteLine("{0} {1}", ex, ex.Message);
                Console.ReadKey();
            }

            Console.ReadKey();
        }
    }
}

File: sample/Python/kabusapi_symbolname_option.py
import urllib.request
import json
import pprint

url = 'http://localhost:18080/kabusapi/symbolname/option'
params = { 'DerivMonth': 202012, 'PutOrCall': 'C', 'StrikePrice': 24000 }

req = urllib.request.Request('{}?{}'.format(url, urllib.parse.urlencode(params)), method='GET')
req.add_header('Content-Type', 'application/json')
req.add_header('X-API-KEY', 'ed94b0d34f9441c3931621e55230e402')

try:
    with urllib.request.urlopen(req) as res:
        print(res.status, res.reason)
        for header in res.getheaders():
            print(header)
        print()
        content = json.loads(res.read())
        pprint.pprint(content)
except urllib.error.HTTPError as e:
    print(e)
    content = json.loads(e.read())
    pprint.pprint(content)
except Exception as e:
    print(e)
